invenio_communities/templates/semantic-ui/invenio_communities/settings.html

```diff
@@ -49,6 +49,9 @@
   <div class="thirteen wide column">
     <div id="app"></div>
   </div>
-  <div id="is_new_community" data-config='{{ False | tojson }}'></div>
+  <div
+    id="form_config"
+    data-is_new='{{ False | tojson }}'
```

Comment:
```suggestion
    data-is-new='{{ False | tojson }}'
```

Which translates to `container.dataset.isNew`